packages/react-refresh/src/ReactFreshRuntime.js

```diff
@@ -178,6 +178,15 @@ function cloneSet<T>(set: Set<T>): Set<T> {
   return clone;
 }
 
+// This is a safety mechanism to protect against rogue getters and Proxies.
+function getProperty(object, property) {
+  try {
+    return object[property];
+  } catch (err) {
+    return undefined;
```

Comment:
Maybe add `// Intentionally ignore.` as a comment before the return.

Reply:
Done.